src/components/LeftPanel.tsx

```diff
@@ -0,0 +1,18 @@
+import { Box, Stack } from "@chakra-ui/react";
+import Profile from "./Profile";
+import SectionsList from "./SectionsList";
+
+const LeftPanel = () => {
+  return (
+    <Stack height="100%" id="left-panel-container">
+      <Box mb={10} id="profile-container" pt={8}>
+        <Profile />
+      </Box>
+      <Box id="list-container">
```

Comment:
is it required to wrap sectionList in Box element?

Reply:
Not hard and fast required, but for consistency and readability, and as we discussed, I have kept this.